### DISCUSSION TITLE
Sequence search in jb2

### DISCUSSION THREAD

**Discussion by anatketem** [GMOD/jbrowse-components#2651] [Q&A]
Hi, 

Is it possible to search for a sequence (not annotation) in Jbrowse2? Is it done by adding sequence search track as in JBrowse1?  

Thanks,

Anat

**Comment by cmdcolin** [comment]
We don't have the search sequence feature yet but definitely this is something we want to do. Ideally we want to get a whole genome BLAT/BLAST type feature integrated (The jbrowse 1 sequence search track is only in your visible region)

**Reply by AimerGDiaz** [reply]
However regular expressions are harder to include as blast queries, I think both options are complementary in principle.

**Comment by yenmr** [comment]
Hi Developer,

I am also interested in the sequence search functionality in JBrowse 2. Is there any possibility of incorporating it in a manner similar to JBrowse 1?

Thank you very much!

Ming-Ren

**Reply by cmdcolin** [reply]
Hi, not sure if you're asking for something different but the basic "sequence search" functionality from jbrowse 1 is available in jbrowse 2.

see screenshots

![image](https://github.com/user-attachments/assets/67e6ec13-e4dc-4945-804b-d9634b5b1904)

![image](https://github.com/user-attachments/assets/23c67ea8-7d72-4493-aef7-2d7847e1786f)

this feature searches the "visible region" for the query and just uses basic exact or regex sequence match

**Reply by yenmr** [reply]
Thank you so much. That is exactly what I am looking for.
MingRen
